decidim/decidim
# Backport smtp settings correction

#### :tophat: What? Why?
When changing fields in `System > Organization > <your-organization> > SMTP Settings` (`from_email` and `from_label`), it does not update.

What's the matter ?
There is three fields concerned by such issue : `from_email`, `from_label` and `from`
`from_email` stores the **bare bone** email of the system ie. `noreply@example.org`
`from_label` stores the **name** under which we should send mails. As instance : 'My organization'
The `from` attribute is there to make a concatenation of both. It is defined in `set_from` in `update_organization_form.rb`

What does this pull request is : 
- correct the hash definition of `smtp_settings` from a string-based-key hash to a symbol-based-key hash
- correct tests that comes along

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?

#### Testing
- Go to `System > Organizations`, edit your organization and change field `from email` and `from label`
- Open a `letter_opener` : `http://localhost:3000/letter_opener`
- Simulate a lost password / a procedure that sends email
- check on `letter_opener` tab that the email hasn't changed

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
